src/AppRoutes.tsx

```diff
@@ -48,11 +48,7 @@ const AppRoutes = () => {
         <Route exact path="/profile" render={(props) => <ProfilePage id={user.id} {...props} />} />
         <Route exact path="/privacy-policy" component={PrivacyPolicy} />
         {/* CreatorRoutes */}
-        <Route
-          exact
-          path="/registration"
-          render={(props) => <EditResourcePage id={user.id} {...props} key={uuidv4()} />}
-        />
+        <Route exact path="/registration" render={(props) => <EditResourcePage id={user.id} {...props} />} />
```

Comment:
Du får se om du klarer å finne noen bug ved å hive ut key'en. Kan den ha vært introdusert fra den tida Routsene lå inni if-setninger istedet for inni HOCs?

Reply:
Ser ut som om det funker. Jeg ser vi hadde en diskusjon 19 nov. i fjor,så det stemmer nok det du sier.

Reply:
"(la til endringene for å tvinge fram refresh av siden ved linking)"